integration/build_test.go

```diff
@@ -108,6 +115,92 @@ func TestBuild(t *testing.T) {
 	}
 }
 
+//see integration/testdata/README.md for details
+func TestBuildInCluster(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+	if !ShouldRunGCPOnlyTests() {
+		t.Skip("skipping test that is gcp only")
+	}
+
+	cleanupSkaffoldBinary := copySkaffoldBinary(t)
+	defer cleanupSkaffoldBinary()
+
+	suffix := uuid.New().String()
+	podName := fmt.Sprintf("skaffold-in-cluster-%s", suffix)
+	cleanupKustomization := setupSuffixKustomization(suffix, t)
+	defer cleanupKustomization()
+
+	const namespace = "default"
```

Comment:
I guess for this exceptional case where additional static resources are needed, using the default namespace is fine.